### PR TITLE
anime cheatsheet

### PR DESCRIPTION
**Pull request by sanket305**
What does your Instant Answer do?

Provides all the information about anime

Why did you choose this data source?

It provides best sites about anime and manga

Are there any other alternative (better) data sources?

Yes, But this is the official one

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

manga and anime lover 

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

No

Are you having any problems? Do you need our help with anything?

NO

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
screenshot 2

### CODE REVIEW

File: share/goodie/cheat_sheets/naruto.JSON [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/naruto.JSON]: Can you delete this file from the PR?

**Comment by mintsoft** [comment]
Hiya @sanket305 Thanks for the PR, can you add the link to the IA page into the description? The Dax bot should have commented in with the links, but they are:
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+   "id":"anime_cheat_sheet",
+   "name":"anime cheat sheet",
+   "description":"all about anime",
+   "metadata":{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: Is there no website where users can go to find more info on these resources or similar information?

Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: If not, we should remove the `metadata`

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: Can you please lowercase the `json` file extension? `anime.json`

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+   "id":"anime_cheat_sheet",
+   "name":"anime cheat sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: Let's chance this to `Anime` or something like `Where to watch, read and download Anime`

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+   "description":"all about anime",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: This should explain what the cheat sheet does: "Provides links to places where you can watch, read and download anime"

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      "sourceName":"DuckDuckHack",
+      "sourceUrl":"https://duck.co/duckduckhack/"
+   },
+   "aliases":[
+      "manga",
+      "anime and manga",
+      "anime&manga",
+      "anime to watch"
+   ],
+   "template_type":"reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: We actually have a special `links` template type which allows you to define a 3rd property for a URL. Then the links become clickable in the cheat sheet!

Please checkout https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/nodejs-tutorials.json

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+   "section_order":[
+      "TO read",
+      "TO watch",
+      "TO download"
+   ],
+   "sections":{
+      "TO read":[
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: Let's change this to something more like "Read Manga Online"

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+         {
+            "val":"http://www.mangahere.co/",
+            "key":"manga here"
+         },
+         {
+            "val":"http://http://mangable.com/",
+            "key":"mangable"
+         }
+      ],
+      "TO watch":[
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: Likewise, "Watch Manga Online"

File: share/goodie/cheat_sheets/json/anime.JSON [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+            "key":"animeShow.tv"
+         },
+         {
+            "val":"http://www.crunchyroll.com/",
+            "key":"crunchyroll"
+         }
+      ],
+      "TO download(use app only)":[
+         {
+            "val":"https://duck.co/ia?repo=goodies",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/anime.JSON]: I'm not sure why these link to the repos?

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Pull request closed without merging**